javascripts/discourse/components/topic-in-gated-category.js

```diff
@@ -6,9 +6,12 @@ const enabledCategories = settings.enabled_categories
   .map((id) => parseInt(id, 10))
   .filter((id) => id);
 
+const enabledTags = settings.enabled_tags.split("|");
```

Comment:
If the setting is an empty string, and we call `.split("|")`, it'll end up as an array like

```
[""]
```
which is kinda weird (and breaks the `.length === 0` check later). We can filter out empty values using the same technique as the `enabledCategory` variable above

```suggestion
const enabledTags = settings.enabled_tags.split("|").filter((t) => t);
```